examples/ds-client.cpp

```diff
@@ -16,29 +16,32 @@
 #include <exception>
 #include <iostream>
 
-#include <deepstream.hpp>
-#include <deepstream/poco/ws.hpp>
+#include <deepstream/core.hpp>
```

Comment:
the examples include poco manually at the moment because the wrapper is kind-of half-baked. ie.
```
#include <deepstream/core.hpp>
#include <deepstream/lib/poco-ws.hpp>
#include <deepstream/lib/basic-error-handler.hpp>
```